pkg/jobmanager/status.go

```diff
@@ -127,26 +131,12 @@ func (jm *JobManager) status(ev *api.Event) *api.EventResponse {
 		JobReport:   report,
 	}
 
-	// Fetch the ID of the last run that was started
-	runID, err := jm.jobRunner.GetCurrentRunAsync(ev.Context, jobID)
+	jobStatus.RunStatuses, err = jm.jobRunner.BuildRunStatuses(ctx, currentJob)
```

Comment:
This is the "sync" codepath (this is a terrible way to name it and I still accept better suggestions).

Previously, this was using `GetCurrentRunAsync`, which talks to a separate DB instance that can point to a replica. There might be replication lag, but that is ok for the status call. We should keep this behaviour, this was done because computing the Status can require a bunch of DB round trips, a replicate might be closer and faster, and it also removes load from the primary instance.